bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```
@@ -388,7 +388,7 @@ spec:
           secret:
             secretName: {{ include "postgresql-ha.postgresqlSecretName" . }}
             items:
-              - key: postgresql-password
+              - key: password
                 path: pgpool-password
```

There is something to take into account here, if `postgresql-ha.postgresqlUsername` is not `postgres`, we need to use `postgresql-password` instead of `password` as you can see in the secret

https://github.com/bitnami/charts/blob/main/bitnami/postgresql-ha/templates/postgresql/secrets.yaml#L17

In order to fix this issue, I'd use the following code

```suggestion
              {{- if eq (include "postgresql-ha.postgresqlUsername" .) "postgres" }}
              - key: password
                path: pgpool-password
              {{- else }}
              - key: postgresql-password
                path: pgpool-password
              {{- end }}             
```

Can you try these changes and confirm they work as expected? 

Thanks